README.md

```diff
@@ -26,8 +26,12 @@ Go here to [create an issue](https://github.com/newrelic/docs-website/issues/new
 
 ### Edit a page
 
-If you'd like to get more directly involved, you can make some changes yourself. On any doc, click the **Edit this page** button to get started making a change.
-You can edit a doc directly on the GitHub site. You don't need to actively fork the site or build it locally to help out. This is the easiest way to contribute.
+If you'd like to get more directly involved, you can edit the docs yourself! Here's how:
+
+1. Every doc page on docs.newrelic.com has an **Edit this page** button. Click it to get started making a change. This will take you to GitHub and the source file for that doc page. 
+2. Edit the file by clicking on the pencil icon. Make your changes, then click **Commit changes**. This will automatically create a fork in your GitHub account with the changes.
```

Comment:
I ran into an issue, somehow, that I was on another branch when I visited the source file. So, I couldn't edit at all. 

I had to be on the `develop` branch and that occurred to me after some lucky circumstances. 

🤷  It might happen others. Up to you to call out that the branch of the source file must be `develop`

Reply:
That's a weird one. Let me know if it keeps happening and we can dig in and try to diagnose.